packages/rest/src/rest-server.ts

```diff
@@ -198,7 +198,7 @@ export class RestServer extends Context implements Server {
     if (this._httpHandler) return;
 
     this._httpHandler = new HttpHandler(this);
-    for (const b of this.find('controllers.*')) {
+    for (const b of this.findByTag('controller')) {
```

Comment:
This is a breaking change, we need to be careful. Please check that our documentation on loopback.io does not mention `controllers.*` pattern anywhere - otherwise the doc will get out of sync with the actual implementation.